### ISSUE TITLE
 Registry layout version 2 is too new to understand

### ISSUE THREAD

**Issue by gmarti** [oxalica/nocargo#4]
Tried to use serde_with 2.0.0 in Cargo.toml

I got the error : "Registry layout version 2 is too new to understand"

Seems there is a new `layout` in cargo ?

serde_with 1.4.0 doesn't have the issue
 
Thanks

**Comment by oxalica** [comment]
Yes, there's a new syntax added recently for specifying features, which uses `crate?/feat` and `dep:crate`.

https://rust-lang.github.io/rfcs/3143-cargo-weak-namespaced-features.html
